tests/unit/utils/test_utils.py

```diff
@@ -1,10 +1,11 @@
 import re
 import os
 
+import mock
```

Comment:
```suggestion
from unittest import mock
```

Reply:
Are we moving from `mock` to `unittest.mock`? cc @efiop 

But, it'd be good to not introduce here?

Reply:
@pared Or even better, just use `mocker` fixture, that would be the most correct way to go about it.

Reply:
@skshetry Even better, we are moving to pytest, so should simply use mocker fixture. Great catch!